justfile

```diff
@@ -50,8 +50,8 @@ _target := if os + '-' + arch == "linux-amd64" {
 _cargo := 'just-cargo profile=' + profile + ' target=' + _target + ' toolchain=' + toolchain
 
 _target_dir := "target" / _target / profile
-_target_bin := _target_dir / "linkerd2-proxy"
-_package_name := "linkerd2-proxy-" + package_version + "-" + arch + (if libc == 'musl' { '-static' } else { '' }) + (if os == 'windows' { '.exe' } else { '' })
+_target_bin := _target_dir / "linkerd2-proxy" + if os == 'windows' { '.exe' } else { '' }
+_package_name := "linkerd2-proxy-" + package_version + "-" + os + "-" + arch + if libc == 'musl' { '-static' } else { '' }
```

Comment:
I am not too sure about this one. The reason being is that it will change the format of the artifacts produced on release. This will also necessitate changes in any scripts that operate on these artifacts (i.e. `fetch-proxy`). Possible options: 
- go an update the scripts in other repos
- add the os param only for windows artifacts

Reply:
this sounds right to me